packaging/installer/netdata-uninstaller.sh

```diff
@@ -65,6 +64,29 @@ if [[ $EUID -ne 0 ]]; then
 	fi
 fi
 
+# -----------------------------------------------------------------------------
+# portable service command
+
+service_cmd="$(command -v service 2>/dev/null)"
+rcservice_cmd="$(command -v rc-service 2>/dev/null)"
+systemctl_cmd="$(command -v systemctl 2>/dev/null)"
+service() {
```

Comment:
```suggestion
portable_service() {
```

Reply:
This was copied directly from `functions.sh`. I intentionally did not change the name to make it trivial to check that the code is the same.